File: 1365-how-many-numbers-are-smaller-than-the-current-number/1365-how-many-numbers-are-smaller-than-the-current-number.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var smallerNumbersThanCurrent = function(nums) {
    let arr=[]
       let count=0
    for (i of nums){
        for(j of nums){
            if(i > j){
                count++
               
            }
             
        }
        arr.push(count)
        count=0
    }
    return arr
};

File: 1365-how-many-numbers-are-smaller-than-the-current-number/1365-how-many-numbers-are-smaller-than-the-current-number.go
func smallerNumbersThanCurrent(nums []int) []int {
    arr:=[]int{}
    for _,val1:= range nums{
        count:=0
        for _,val2:= range nums{
            if val1 > val2{
                count++
            }
        }
        arr=append(arr,count)
    }
    return arr
}